boston-housing/index.js

```diff
@@ -51,28 +51,56 @@ export const arraysToTensors = () => {
  *
  * @returns {tf.Sequential} The linear regression model.
  */
-export const linearRegressionModel = () => {
+export function linearRegressionModel() {
   const model = tf.sequential();
   model.add(tf.layers.dense({inputShape: [bostonData.numFeatures], units: 1}));
+
+  model.summary();
   return model;
 };
 
 /**
  * Builds and returns Multi Layer Perceptron Regression Model
- * with 2 hidden layers, each with 10 units activated by sigmoid.
+ * with 1 hidden layers, each with 10 units activated by sigmoid.
  *
  * @returns {tf.Sequential} The multi layer perceptron regression model.
  */
-export const multiLayerPerceptronRegressionModel = () => {
+export function multiLayerPerceptronRegressionModel1Hidden() {
+  const model = tf.sequential();
+  model.add(tf.layers.dense({
+    inputShape: [bostonData.numFeatures],
+    units: 50,
+    // activation: 'sigmoid',
```

Comment:
I was seeing similar results for linear regressor and MLP with 1 hidden layer. This should be uncommented to fix that.

Reply:
Good catch. Done.